app/jobs/characterize_job.rb

```diff
@@ -6,16 +6,28 @@ class CharacterizeJob < ApplicationJob
   # @param [FileSet] file_set
   # @param [String] file_id identifier for a Hydra::PCDM::File
   # @param [String, NilClass] filepath the cached file within the Hyrax.config.working_path
```

Comment:
```suggestion
  # @param [String, NilClass] filepath the cached file within the Hyrax.config.working_path
  # @param [String, NilClass] delete_characterization_path cleanups parent directory when set to 'include_parent_dir'
```

Reply:
@dlpierce Oh, snap, I skipped that!  Addressed in new commit